src/WebJobs.Script/Content/Script/functions.js

```diff
@@ -16,6 +15,22 @@ function globalInitialization(context, callback) {
     process.on('uncaughtException', function (err) {
         context.handleUncaughtException(err.stack);
     });
+
+    if (context.console) {
+        var capture = (stream) => {
+            // ensure that stream has not already been wrapped (edge sets stdout and stderr to same backing NullStream object)
```

Comment:
Not already wrapped by us. 

Since edge points stdout & stderr to the same object there was the duplicate logging behavior.  This ensures that logging will work regardless of whether stdout & stderr are different objects (the normal case) or the same object (the edgejs case).